packages/frontend-2/components/project/page/models/Card.vue

```diff
@@ -1,17 +1,34 @@
+<!-- eslint-disable vuejs-accessibility/click-events-have-key-events -->
```

Comment:
See comment below on StructureItem. keyboardClick has been deprecated and replaced with a directive which does not appease eslint.

Reply:
@andrewwallacespeckle Yea, I created the directive and deprecated the utility, but ESLint not picking up on it is a real problem. I guess it's cleaner to just keep using the utility (and un-deprecate it?) so that the ESLint rule isn't disabled for the entire file, cause you could miss other elements that need the same thing.

Its up to you which route you go for - the directive or the utility, but make sure some kind of keyboard handling is there.